etc/docker-scripts/bootstrapper.py

```diff
@@ -189,11 +189,14 @@ def execute(self) -> None:
         notebook_name = notebook.replace('.ipynb', '')
         notebook_output = notebook_name + '-output.ipynb'
         notebook_html = notebook_name + '.html'
+        kernel_name = "python3"
 
         try:
             OpUtil.log_operation_info(f"executing notebook using 'papermill {notebook} {notebook_output}'")
             t0 = time.time()
-            subprocess.run(['papermill', notebook, notebook_output], check=True)
+            # Really hate to do this but have to invoke Papermill via library as workaround
+            import papermill
+            papermill.execute_notebook(notebook, notebook_output, kernel_name=kernel_name)
```

Comment:
This should be fine.  We should remove `kernel_name` from the arguments since it will be [pulled from the notebook](https://github.com/nteract/papermill/blob/main/papermill/execute.py#L91) if not specified (and remove the comment :smile: ).